### ISSUE TITLE
Request for Show On Start tick box for Trigger Zone 

### ISSUE THREAD

**Issue by synchromesh62** [TheGameCreators/GameGuruRepo#2772]
As the heading states there are times i could have really used a trigger zone to trigger another like the Win Zone for example.
Imagine the Win Zone does not appear until you have got to a radio to request an extraction point that may be in plain sight.
The radio Trigger zone around the radio would make the Win Zone appear once you have carried out that task.
So many uses for a simple feature :)

**Comment by Maro32** [comment]
+1
I'll appreciate a tick box for trigger zones.

**Comment by mike123yy** [comment]
Yeah, if you're going to make anything doesn't direct you on a path straightforward this is definitely needed. I have a game I'm working on that needs this. Otherwise I have to make all the events happen from point a to point B instead of making the player go somewhere, triggering event and then go back and another event can be triggered. In an indoor location it kind of ruins the experience of exploring or doing tasks
